spec/unit/settings_spec.rb

```diff
@@ -1377,8 +1377,6 @@ def assert_accessing_setting_is_deprecated(settings, setting)
     describe "when adding users and groups to the catalog" do
       before do
         Puppet.features.stubs(:root?).returns true
-        Puppet.features.stubs(:microsoft_windows?).returns false
```

Comment:
We'll have to look at this one more closely... this one might be a can of worms.